test/run.sh

```diff
@@ -142,22 +141,22 @@ else
   do
     TEST_NAME=$(basename "${TEST_PATH}")
     LOG_FILE="${LOGS_DIR}/${TEST_NAME}.log"
-    echo "running integration for ""${TEST_NAME}"" - log file at ${LOG_FILE}" | werft log slice "test-${TEST_NAME}"
+    echo "running integration for ""${TEST_NAME}"""
 
     cd "${TEST_PATH}"
     set +e
-    go test -v ./... "${args[@]}" 2>&1 | tee "${LOG_FILE}" | werft log slice "test-${TEST_NAME}"
+    go test -v ./... "${args[@]}" 2>&1 | go-junit-report -subtest-mode=exclude-parents -set-exit-code -out "results/TEST-${TEST_NAME}.xml" -iocopy
     RC=${PIPESTATUS[0]}
     set -e
     cd -
 
     if [ "${RC}" -ne "0" ]; then
       FAILURE_COUNT=$((FAILURE_COUNT+1))
-      werft log slice "test-${TEST_NAME}" --fail "${RC}"
-    else
-      werft log slice "test-${TEST_NAME}" --done
     fi
   done
 fi
 
+xunit-viewer -r ./results -o test-output.html
+pkill -f "port-forward"
```

Comment:
Sometimes after tests are done the port-forwards linger, and if there's too many, I've seen the workspace be stopped